gradle_intro/README.md

```diff
@@ -10,11 +10,12 @@ The goal of this exercise is to successfully download the latest version of a de
 ## Exercise
 
 1. Navigate to your exercise folder
-1. Add your gradle repository to `build.gradle`
+1. Add your gradle repository to `build.gradle` Go to Artifactory and navigate to your $KATA_USERNAME-generic-gradle-1 repository.
++    Copy URL to file path from the 'General' page
 
     ```groovy
     repositories {
-        maven { url "http://ArtifactoryUrl.com/artifactory/$KATA_USERNAME-generic-gradle-1" }
+        maven { url "<Copied from your Artifactory $KATA_USERNAME-generic-gradle-1 repository General page>" }
```

Comment:
I think this part is hard to follow, as I (as a student) am uncertain if all text in <> should be exchanged or not.
Possible solution:
Only write `copied Artifactory URL` in the <> section

Reply:
Hmm, it is not Artifactory URL (https://artifactoryurl.com/artifactory) that is needed here, but the repo link.
Maybe
`maven { url "<Copied from your Artifactory repository General page>" } `
would convey the meaning better?

Reply:
I think so too yes :)